src/MetaModels/Attribute/Select/MetaModelSelect.php

```diff
@@ -54,7 +54,7 @@ class MetaModelSelect extends AbstractSelect
     protected function checkConfiguration()
     {
         return parent::checkConfiguration()
-            && (null !== $this->getSelectMetaModel());
+               && (null !== $this->getSelectMetaModel());
```

Comment:
Have to change this. Copy&Paste mistake.

Reply:
@zonky2 he is already in, see: https://github.com/MetaModels/attribute_select/pull/62/files#diff-31dffa8cdd5fd0725c2fd39310f1fa6bR16

Reply:
Oh sorry, you are right, his family name is in lowercase, should be changed.